### ISSUE TITLE
Changing the size of tap target from 48 px to 44 px

### ISSUE THREAD

**Issue by ptu14** [GoogleChrome/lighthouse#14879]
## Main question

Can we change the minimum clickable area from [48 px](https://github.com/GoogleChrome/lighthouse/blob/main/core/audits/seo/tap-targets.js#L45) to 44 px? It doesn't make sense to penalize users with lower seo ranking if they make their sites and apps according to[ WCAG guidelines](https://www.w3.org/TR/WCAG21/#target-size).

## Consideration

Is there still a point in maintaining these two values separately? 

I did a little research and the only place where `48 px` appears is on Google materials, but apart from the old android documentation pointed out by @paulirish there are no sources. Probably this value is copied and no one even wonders why.

WCAG is based on, among other things, this article: [Target Size Study for One-Handed Thumb Use on Small Touchscreen Devices](http://www.cs.umd.edu/hcil/trs/2006-11/2006-11.htm)

>Along with subjective ratings and the findings on hit response variability, we found that target size of 9.2 mm for discrete tasks and targets of 9.6 mm for serial tasks should be sufficiently large for one-handed thumb use on touchscreen-based handhelds without degrading performance and preference.

`9.6 mm` is about `36 px`

but the value taken from this scientific study is `11.5mm` which after [conversion](https://www.w3.org/TR/css3-values/#absolute-lengths) is about `43,47 px`  which, when rounded up, gives `44 px`.

I searched for more recent studies, but found nothing.

If one wanted to be very accurate, even the android keyboard does not meet this requirement on small screens, because the line containing **q,w,e,r,t,y,u,i,o,p** has 10 keys, so on `360 px` screens there is only `36 px` touch width for one key. 

## Other links

here Microsoft menitions 7.5mm square range (40x40 pixels on a 135 PPI display at a 1.0x scaling plateau): https://learn.microsoft.com/en-us/windows/apps/design/input/guidelines-for-targeting

old Apple doc 44px x 44px: https://web.archive.org/web/20170805151945/https://developer.apple.com/ios/human-interface-guidelines/visual-design/layout/

<div type='discussions-op-text'>

<sup>Originally posted by **ptu14** March  7, 2023</sup>
Hi,

why the minimum clickable area is [48x48](https://github.com/GoogleChrome/lighthouse/blob/main/core/audits/seo/tap-targets.js#L45)?

[WCAG 2.1 (level AAA) target-size](https://www.w3.org/TR/WCAG21/#target-size) it specifies the minimum value as 44x44 CCS pixels.

[draft WCAG 2.2 (level AA) target-size](https://www.w3.org/TR/WCAG22/#target-size-minimum) goes even further and specifies it as 24x24. But it's still a draft for now.

from what i understand, your value is subjectively set and has nothing to do with WCAG?

does it just mean if you want a higher google search result, do larger clickable areas than WCAG dictates?</div>

**Comment by connorjclark** [comment]
ref: https://github.com/GoogleChrome/lighthouse/issues/12345#issuecomment-1021786442

**Comment by oliviertassinari** [comment]
I wonder if the rule shouldn't also factor in the touch area. For example, a 182x29 link footer seems simple enough to tap on, but a 50x29 wouldn't be OK.

**Comment by adamraine** [comment]
#12345 best captures our work to align our tap targets advice with current a11y guidelines.
